web/src/app/administration/components/access-items-management-dialog/access-items-management-dialog.component.html

```diff
@@ -0,0 +1,15 @@
+<h4>Belonging groups</h4>
+<mat-dialog-content>
+    <mat-list *ngIf="groups && groups.length > 0; else no">
+        <mat-list-item *ngFor="let group of groups">
+            {{group.name}}
+            <mat-divider></mat-divider>
+        </mat-list-item>
+    </mat-list>
+    <ng-template #no>The user is not associated to
+        any groups
+    </ng-template>
+</mat-dialog-content>
+<mat-dialog-actions align="end" autofocus="false">
```

Comment:
The component is actually not used. I just didn't remove it yet, since we could need it in the future :) But I can anyway remove the align parameter. I'm not sure what you mean with the second annotation...

Reply:
ah okay, I didn't know that is isn't used.